tools/llm_bench/llm_bench_utils/memory_monitor.py

```diff
@@ -399,17 +336,84 @@ def get_data(self):
         return (self.memory_data['full_mem'].get(MemoryType.RSS, -1), self.memory_data['from_zero'].get(MemoryType.RSS, -1),
                 self.memory_data['full_mem'].get(MemoryType.SYSTEM, -1), self.memory_data['from_zero'].get(MemoryType.SYSTEM, -1))
 
-    def log_data(self, comment):
-        max_rss_mem, max_rss_increase, max_sys_mem, max_sys_increase = self.get_data()
-        msg = (f"Max rss memory cost {comment}: {max_rss_mem:.2f}{self.memory_unit.value}, "
-               f"rss memory increase {comment}: {max_rss_increase:.2f}{self.memory_unit.value}, "
-               f"max system memory cost {comment}: {max_sys_mem:.2f}{self.memory_unit.value}, "
-               f"system memory increase {comment}: {max_sys_increase:.2f}{self.memory_unit.value}")
+
+class MemStatus():
+    def __init__(self, rss=None, sys=None):
+        self.rss = rss
+        self.sys = sys
+
+
+class MemoryDataCollector():
```

Comment:
`Collector` seems to be too general name. Every nested Memory* class name could have that word. Maybe `Summarizer` is a better word to give a hint about hierarchy